src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ObjectWriter.cs

```diff
@@ -19,7 +19,7 @@ namespace ILCompiler.DependencyAnalysis
     /// <summary>
     /// Object writer using https://github.com/dotnet/llilc
     /// </summary>
-    internal class ObjectWriter : IDisposable
+    public class ObjectWriter : IDisposable
```

Comment:
The change is no longer needed. Now we compile coreRT and the hosting code into one module.